courses/models.py

```diff
@@ -263,6 +263,17 @@ def can_freeze_grades(self):
             raise ImproperlyConfigured('Missing freeze_grade_date')
         return datetime.now(pytz.utc) > self.freeze_grade_date
 
+    @property
+    def has_frozen_grades(self):
+        """
+        Checks if the grades for the course have been frozen
+        """
+        try:
+            freeze_status = self.courserungradingstatus
+        except self._meta.model.courserungradingstatus.RelatedObjectDoesNotExist:
```

Comment:
no, it would require to import `CourseRunGradingStatus` triggering a circular import error